test/parallel/test_tensorflow.py

```diff
@@ -53,9 +53,11 @@ class TensorFlowTests(BaseTensorFlowTests):
     """
     def __init__(self, *args, **kwargs):
         super(TensorFlowTests, self).__init__(*args, **kwargs)
+        if hvd.sycl_built():
+            os.environ['ITEX_LIMIT_MEMORY_SIZE_IN_MB'] = '4096'
 
     def test_gpu_required(self):
-        if not tf.test.is_gpu_available(cuda_only=True):
+        if not tf.test.is_gpu_available(cuda_only=True) and not hvd.sycl_built():
```

Comment:
So tensorflow thinks, there is no GPU available, but there is when sycl is built?